docs/channels/library_auth_reference/rest-api.md

```diff
@@ -57,7 +57,7 @@ The event data should not be larger than 10KB. If you attempt to POST an event w
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name      | Event name (required)                                                                                                                                                                                                                                    |
 | data      | Event data (required) - limited to 10KB                                                                                                                                                                                                                  |
-| channels  | Array of one or more channel names - limited to 100 channels                                                                                                                                                                                             |
+| channels  | Array of one or more channel names - limited to 100 channels. When using encrypted channels this parameter is not supported.                                                                                                                                                                                             |
```

Comment:
This is not true. `channels` is an array. That's fine if user wants to use it for encrypted channels. It just has to be the only name on the list. We can leave this line unchanged. Info about encrypted channels will remain within encrypted channels page (no duplicates) and the API ref is cleaner.

Reply:
Fixed, thanks!